lib/iris/__init__.py

```diff
@@ -113,6 +113,7 @@ def callback(cube, field, filename):
 from iris._deprecation import IrisDeprecation, warn_deprecated
 import iris.fileformats
 import iris.io
+import iris.options
```

Comment:
I disagree; I think this _is_ a good thing. We want to make `options` as easy to use as possible. It makes complete sense that the controlling options for Iris come as part of the primary import.

And it's not just me that thinks so -- dask does this, as does `iris.FUTURE`, by dint of being defined in the Iris `__init__` already. And adding `options` to the `__init__` actually was a design consideration I made, which I only dropped because of the potential size of the `options` module.